polytope_server/common/datasource/fdb.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, config):
         os.environ["FDB_HOME"] = self.config.get("fdb_home", "/opt/fdb")
         import pyfdb
 
-        self.fdb = pyfdb.FDB()
+        pyfdb.FDB()
```

Comment:
Since this is a workaround that is separate from the Sentry instrumentation, I think we want to revert the FDB part of this pull request in the future.

Given that, I would recommend
1. Split this pull request into sentry instrumentation and fdb workaround
2. Add a TODO here to revert the workaround PR after we resolve the issue with hanging connections
3. Open a jira issue to track the revert

I haven't seen any TODOs in our codebase so maybe that isn't a practice and just having a jira issue noting which pull request should be reverted is sufficient. Putting it in a separate PR allows a clean revert that ensures we didn't miss anything or accidentally undo some of the sentry instrumentation.